extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,510 @@
+
+<img src="figures/placemarks.png"  alt="Placemarks" width="600">
+
+```jsonc
+{
+  "primitives": [
+    {
+      "attributes": {
+        "POSITION": 0
+      },
+      "mode": 0,
+      "extensions": {
+        "EXT_feature_metadata": {
+          "featureIdAttributes": [
+            {
+              "featureTable": "placemarks",
+              "featureIds": {
+                "constant": 0,
+                "divisor": 1
+              }
+            }
+          ]
+        }
+      }
+    }
+  ]
+}
+```
+### Feature ID Textures
+
+Sometimes it is helpful to classify the pixels of an image into different features. Some examples include segmenting an image in a computer vision project or marking regions on a map.
+
+Often per-texel feature IDs provide finer granularity than per-vertex feature IDs like in the example below.
+
+<img src="figures/feature-id-texture.png"  alt="Feature ID Texture" width="800">
+
+```jsonc
+{
+  "primitives": [
+    {
+      "attributes": {
+        "POSITION": 0,
+        "TEXCOORD_0": 1
+      },
+      "indices": 2,
+      "material": 0,
+      "extensions": {
+        "EXT_feature_metadata": {
+          "featureIdTextures": [
+            {
+              "featureTable": "buildingFeatures",
+              "featureIds": {
+                "texture": {
+                  "texCoord": 0,
+                  "index": 0
+                },
+                "channels": "r"
+              }
+            }
+          ]
+        }
+      }
+    }
+  ]
+}
+```
+
+`texture` is a glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `channels` must be a single channel (`"r"`, `"g"`, `"b"`, or `"a"`). Furthermore, feature IDs must be whole numbers in the range `[0, count - 1]`, where `count` is the total number of features in the feature table. Texture filtering should be disabled when accessing feature IDs.
+
+### Feature ID Instance Attributes
+
+Feature IDs may also be assigned to individual instances when using the [`EXT_mesh_gpu_instancing` extension](../../EXT_mesh_gpu_instancing/README.md). This works the same way as assigning feature IDs to vertices. Feature IDs may be stored in accessors or generated implicitly.
+
+```jsonc
+{
+  "nodes": [
+    {
+      "mesh": 0,
+      "extensions": {
+        "EXT_mesh_gpu_instancing": {
+          "attributes": {
+            "TRANSLATION": 0,
+            "ROTATION": 1,
+            "SCALE": 2,
+            "_FEATURE_ID_0": 
+          },
+          "extensions": {
+            "EXT_feature_metadata": {
+              "featureIdAttributes": [
+                {
+                  "featureTable": "trees",
+                  "featureIds": {
+                    "attribute": "_FEATURE_ID_0"
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+    }
+  ]
+}
+```
+
+## Feature Metadata
```

Comment:
Just wondering, why do we discuss feature metadata before feature identification? While I guess it's somewhat arbitrary, to me it makes more sense if you explain what metadata is, _then_ describe how to reference it via identification.

Reply:
It could be argued that feature identification is more core to the extension because it allows individual features to be identified, which is a perquisite to assigning metadata. There are other operations you can do on features besides using metadata (show/hide/highlight color).

Reply:
But I agree, it's somewhat arbitrary. This organization just felt right this go around, especially in lieu of the updated positioning in the overview.